src/client/js/scripts/RegisterPageErrorHandler.js

```diff
@@ -0,0 +1,27 @@
+export default class RegisterPageErrorHandler {
+  constructor(resultHolder) {
+    this.resultHolder = resultHolder;
+  }
+
+  registerHandle(error) {
+    if (error.name === 'FormatError' && error.message === 'Wrong second password') {
+      this.resultHolder.textContent = 'Два пароля не совпадают';
+    } else if (error.name === 'FormatError') {
```

Comment:
Проверка имён имеет похожие проблемы, что и проверка сообщения. Лучше проверять, является ли ошибка объектом класса. Так повышается типобезопасность.
```suggestion
    } else if (error instanceof FormatError') {
```